src/components/bread-crumb-nav/bread-crumb-nav.js

```diff
@@ -1,42 +1,50 @@
 import { Breadcrumb } from '@fluentui/react/lib/Breadcrumb';
+import { PATHS, ROUTE_NAME_BY_PATH } from 'common/constants';
+import { getSplitPaths } from 'common/functions';
+import { useConversionStore, useUserStore } from 'common/store';
+import { useEffect } from 'react';
 import { useTranslation } from 'react-i18next';
-import { useNavigate, useLocation } from 'react-router-dom';
-
+import { useLocation, useNavigate } from 'react-router-dom';
+import { shallow } from 'zustand/shallow';
 import { breadcrumbStyle } from './bread-crumb-nav.style';
-import { getSplitPaths } from 'common/functions';
-import { PATHS, ROUTE_NAME_BY_PATH } from 'common/constants';
-import { useConversionStore } from 'common/store';
 
-const routesConfirmationRequiredBeforeLeaving = [PATHS.CONVERSION];
+const routesReset = [PATHS.CONVERSION];
+
+const userStoreSelector = s => [s.subscriptionKey];
```

Comment:
minor thing: when selecting a single value it's better not to use arrays but return it directly s => s.subKey.
With this you won't need to add shallow as it is redundant with primitives

Reply:
Fixed in https://github.com/Azure/azure-maps-creator-onboarding-tool/pull/104